package.json

```diff
@@ -34,7 +36,9 @@
   },
```

Comment:
- [ ] `peerDependencies.react` should be updated to include `|| ^16.0.0`.
- [ ] `peerDependencies` should include `"react-dom": "^0.14.3 || ^15.0.0 || ^16.0.0"`